PHP/hw2.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+$n = 10;
+
+function factorialOfNumber($n)
+{
+    $fact = 1;
+    for($i=$n; $i>0; $i--) $fact *= $i;
```

Comment:
1. Please try to follow coding standard at https://www.php-fig.org/psr/psr-2/ 
Let take a look on `if-else` and `for`.
Following coding standard helps your source code more clear.
2. `$i > 0` should be $i > 1,  It will save one loop. Because a number $a*1 equals to $a, so that multiple by 1 is not necessary.